src/js/controller/model.js

```diff
@@ -223,12 +228,26 @@ define([
                 this.set('flashBlocked', false);
             }
 
+            var presetPlaybackRate;
+            // If there is no current provider, then the video tag might have a preset playback rate...
+            if (!_provider) {
+                presetPlaybackRate = _currentProvider.getPlaybackRate();
+            }
+
             _provider = _currentProvider;
             _provider.volume(_this.get('volume'));
 
             // Mute the video if autostarting on mobile. Otherwise, honor the model's mute value
             _provider.mute(this.autoStartOnMobile() || _this.get('mute'));
 
+            if (!presetPlaybackRate) {
+                // Attempt setting the playback rate to be the user selected value
+                this.setPlaybackRate(this.get('defaultPlaybackRate'));
+            } else {
+                // Set to playback rate that was in use before setting up the player.  defaultPlaybackRate is user value
+                this.set('playbackRate', presetPlaybackRate);
```

Comment:
Why do we need this feature? Why not ignore playbackRate before setup.

It seems more likely to prevent a call to `api.setPlaybackRate` from working than anything else.